recipes/nsync/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class NsyncConan(ConanFile):
+    name = "nsync"
+    homepage = "https://github.com/google/nsync"
+    description = "Library that exports various synchronization primitive"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("c", "thread", "multithreading", "google")
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
```suggestion
    settings = "os", "compiler", "build_type", "arch"
    options = {"fPIC": [True, False], "shared": [True, False]}
    default_options = {"fPIC": True, "shared": False}
```

Let's add shared and fPIC

Reply:
It's enabled by default in https://github.com/google/nsync/blob/master/CMakeLists.txt#L18
Should we patch the library to allow turn it off?